client/network.go

```diff
@@ -129,9 +171,22 @@ func (cli *NetworkCli) CmdNetworkInfo(chain string, args ...string) error {
 	if err != nil {
 		return err
 	}
-	if _, err := io.Copy(cli.out, bytes.NewReader(obj)); err != nil {
+
+	networkResource := &networkResource{}
+	if err := json.NewDecoder(bytes.NewReader(obj)).Decode(networkResource); err != nil {
 		return err
 	}
+	fmt.Fprintf(cli.out, "Network Id: %s\n", networkResource.ID)
```

Comment:
I left one just so folks can get a clear visual of a Network ID vs. a Service ID.

```
./dnet network service info foo net1
Service Id: 66ba197307d05f8920d376dae43ef9cf78cb38a9cdbe1d9eeba48ed90b819dee
	Name: foo
	Network: net1

./dnet network service info bar net1
Service Id: b790780bacded89bd90330ea36160a9b005c2abfcfb66e81ae2ece5de8a0b58e
	Name: bar
	Network: net1
 
./dnet network info net1
Network Id: 57b3fab23c33f77ac26fdba767fe084257597ef8bfc16bfb375f26cc39359ce5
Name: net1
Type: null
  Service Id: 66ba197307d05f8920d376dae43ef9cf78cb38a9cdbe1d9eeba48ed90b819dee
	Name: foo
	Network: net1
  Service Id: b790780bacded89bd90330ea36160a9b005c2abfcfb66e81ae2ece5de8a0b58e
	Name: bar
	Network: net1
```
Not attached to it by any means but see what you think with it indented now. I did a 2 space and then a \t tab. I liked your suggestion on 2 layers of indentations. It looks quite clear in a terminal.